app/models/doi.rb

```diff
@@ -408,6 +410,14 @@ class Doi < ApplicationRecord
         updated: { type: :date },
         deleted_at: { type: :date },
         cumulative_years: { type: :integer, index: "false" },
+        subjects: { type: :nested, include_in_parent: true,  properties: {
```

Comment:
Now that the approach is to index FOS separately, could this be switched to being object type rather than nested?

Reply:
Yes